container-toolkit/release-notes.md

```diff
@@ -10,6 +10,53 @@ This document describes the new features, improvements, fixed and known issues f
 
 ______________________________________________________________________
 
+## NVIDIA Container Toolkit 1.17.0
+
+This version includes updates for:
+
+* [NVIDIA CVE-2024-0134](https://nvidia.custhelp.com/app/answers/detail/a_id/5585)
+
+To view any published security bulletins for NVIDIA products, refer to the NVIDIA product security page at <https://www.nvidia.com/en-us/security/>.
+
+For more information regarding NVIDIA security vulnerability remediation policies, refer to <https://www.nvidia.com/en-us/security/psirt-policies/>.
+
+---
+
+This release of the NVIDIA Container Toolkit `v1.17.0` is a feature update and bugfix release.
+
+The following packages are included:
+
+- `nvidia-container-toolkit 1.17.0`
+- `libnvidia-container-tools 1.17.0`
+- `libnvidia-container1 1.17.0`
+
+The following `container-toolkit` conatiners are included:
+
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.17.0-ubi8`
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.17.0-ubuntu20.04` (also as `nvcr.io/nvidia/k8s/container-toolkit:v1.16.1`)
+
+### Fixes and Features
+
+- Fixed a bug where symlinks created when starting a container may escape the container root. This addresses CVE-2024-0134.
+- Fixed a bug when locating libcuda.so in the ldcache when generating CDI specifications. This allows the driver libraries to be properly discovered on systems with non-standard driver installation paths.
+- Fixed a bug in the creation of symlink chains on Tegra-based sytems. This ensures that behaviour is consistent across multiple runs of containers and do not depend on a random sort order.
```

Comment:
nit: "Fixed a bug related to creating symlink chains..."

"This fix ensures..." or some other noun, please.